modules/client/src/connext.ts

```diff
@@ -496,6 +504,31 @@ export class ConnextInternal extends ConnextChannel {
     return await this.conditionalTransferController.conditionalTransfer(params);
   };
 
+  public watchForUserWithdrawal = async (tx: CFCoreTypes.MinimalTransaction): Promise<void> => {
+    // poll for withdrawal tx submitted to multisig matching tx data
+    const maxBlocks = 15; // wait 15 blocks before retrying/self-submitting
+    const startingBlock = await this.ethProvider.getBlockNumber();
+    let found;
+    interval(async (iteration, stop): Promise<void> => {
+      const currentBlock = await this.ethProvider.getBlockNumber();
+      if (currentBlock <= startingBlock) {
+        return;
+      }
+      found = await this.checkForUserWithdrawal(currentBlock, tx);
```

Comment:
Would we want to use ethers subscriptions/event listening for this instead of polling?